src/main/kotlin/com/github/b3er/idea/plugins/arc/browser/base/sevenzip/SevenZipArchiveHandler.kt

```diff
@@ -31,6 +31,11 @@ open class SevenZipArchiveHandler(path: String) :
                 }
             }
         }
+        private fun toArchive(raf: RandomAccessFile): IInArchive? {
+            val archive = SevenZip.openInArchive(null, RandomAccessFileInStream(raf))
+            raf.close()
```

Comment:
Shouldn't you use `use` instead? https://www.baeldung.com/kotlin-try-with-resources